smorest_crud/view.py

```diff
@@ -14,19 +14,32 @@
 
 
 class CRUDView(MethodView):
-    """Base class for collection and resource views."""
+    """Base class for collection and resource views.
+
+    This works like `Flask MethodView <https://flask.palletsprojects.com/en/1.1.x/views/#method-views-for-apis>`_;
+    python methods are named after HTTP verbs.
+    """
 
-    # define these, please
     model: Model
+    """SQLAlchemy model we are providing CRUD services for."""
+
     access_checks_enabled: bool = True
+    """Whether or not to apply access checks to models.
+
+    Requires `CRUD_ACCESS_CHECKS_ENABLED` configuration to be enabled."""
 
-    # default
     decorators = [jwt_required]
+    """List of decorators to apply to view functions.
+
+    Applies `jwt_required <https://flask-jwt-extended.readthedocs.io/en/stable/api/#flask_jwt_extended.jwt_required>`_ by default to required authenticated requests.
+    """
 
     def query(self) -> BaseQuery:
+        """Return query for `model`."""
         return self._get_model().query
 
     def query_for_user(self) -> BaseQuery:
+        """Produce a query for current model, filtered by `model.query_for_user(current_user)`."""
```

Comment:
Backticks do not seem to enable monospace here: 
![image](https://user-images.githubusercontent.com/7350042/82870137-21aa1100-9f38-11ea-9f05-9b27b23e714e.png)

Reply:
I couldn't figure out what it should be

Reply:
Maybe the theme sucks idk